_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -386,13 +386,10 @@ public static function get_plans( $request ) {
 		$data = $use_cache ? get_transient( 'jetpack_plans' ) : false;
 
 		if ( false === $data ) {
-			$path = '/plans';
+			$path = '/plans?_locale=' . get_user_locale();
 			// passing along from client to help geolocate currency
-			$ip = $_SERVER['HTTP_X_FORWARDED_FOR']; // if we already have an list of forwarded ips, then just use that
-			if ( empty( $ip ) ) {
-				$ip = $_SERVER['HTTP_CLIENT_IP']; // another popular one for proxy servers
-			}
-			if ( empty( $ip ) ) {
+			$ip = Jetpack::current_user_ip( true );
+			if ( empty( $ip ) && isset( $_SERVER['REMOTE_ADDR'] ) ) {
```

Comment:
This whole `if` block may not be needed as `current_user_ip()` [does it too](https://github.com/Automattic/jetpack/blob/master/class.jetpack.php#L1757).

Reply:
oh, damn it, I missed that last `return` in the function :man_facepalming: